rclcpp/include/rclcpp/experimental/executors/events_executor/events_executor.hpp

```diff
@@ -92,6 +92,8 @@ class EventsExecutor : public rclcpp::Executor
   void
   spin() override;
 
+  using Executor::spin;
```

Comment:
to be precise, the events executor should have a dedicated implementation.
This line here https://github.com/ros2/rclcpp/blob/070d71e2c68bd8009c7f96c3c110f2cb2390dde3/rclcpp/src/rclcpp/experimental/executors/events_executor/events_executor.cpp#L114 will start a new thread

Reply:
fair point...

Reply:
done